cache.rst

```diff
@@ -0,0 +1,209 @@
+Cache
+=====
+
+Using cache is a great way of making you application run quicker. The Symfony cache
+component is shipped with many adapters to different storages. Every adapter is
+developed for high performance.
+
+Basic uses of the cache looks like this::
+
+    use Symfony\Contracts\Cache\ItemInterface;
+
+    // The callable will only be executed on a cache miss.
+    $value = $pool->get('my_cache_key', function (ItemInterface $item) {
+        $item->expiresAfter(3600);
+
+        // Do some HTTP request or heavy computations
+        $computedValue = 'foobar';
+
+        return $computedValue;
+    });
+
+    echo $value; // 'foobar'
+
+    $pool->delete('some_key');
+
+
+Symfony also supports PSR-6 and PSR-16 cache interfaces. You can read more about
+these at the :doc:`component documentation </components/cache>`.
+
+
+Configuring Cache with FrameworkBundle
+--------------------------------------
+
+When configuring the the cache component there are a few concepts you should know
+of:
+
+**Pool**
+    This is a service that you will interact with. Each pool will always have
+    its own namespace and cache items. There is never a conflict between pools.
+**Adapter**
+    An adapter is a *template* that you use to create Pools.
+**Provider**
+    A provider is the DSN connection to the actual storage.
+
+There are two pools that are always enabled by default. They are ``cache.app`` and
+``cache.system``. The system cache is use for things like annotations, serializer,
+and validation. The ``cache.app`` can be used in your code. You can configure which
+adapter (template) they use by using the ``app`` and ``system`` key like:
+
+.. code-block:: yaml
+
+    # config/packages/cache.yaml
+    framework:
+        cache:
+            app: cache.adapter.filesystem
+            system: cache.adapter.system
+
+The Cache component comes with a series of adapters already created:
+
+* :doc:`cache.adapter.apcu </components/cache/adapters/apcu_adapter>`
+* :doc:`cache.adapter.array </components/cache/adapters/array_cache_adapter>`
+* :doc:`cache.adapter.doctrine </components/cache/adapters/doctrine_adapter>`
+* :doc:`cache.adapter.filesystem </components/cache/adapters/filesystem_adapter>`
+* :doc:`cache.adapter.memcached </components/cache/adapters/memcached_adapter>`
+* :doc:`cache.adapter.pdo </components/cache/adapters/pdo_doctrine_dbal_adapter>`
+* :doc:`cache.adapter.redis </components/cache/adapters/redis_adapter>`
+* :doc:`PHPFileAdapter </components/cache/adapters/php_files_adapter>`
+* :doc:`PHPArrayAdapter </components/cache/adapters/php_array_adapter>`
```

Comment:
This file is called php_array_cache_adapter (consistency problem spotted :smiley: )

Reply:
Thank you